code/game/gamemodes/miniantags/pulsedemon/pulsedemon_objectives.dm

```diff
@@ -0,0 +1,54 @@
+/datum/objective/pulse_demon/infest
+	name = "Hijack APCs"
+	var/amount = 20
+
+/datum/objective/pulse_demon/infest/New()
+	. = ..()
+	amount = rand(3, 5) * 5
+	explanation_text = "Hijack [amount] APCs."
+
+/datum/objective/pulse_demon/infest/check_completion()
+	if(..())
+		return TRUE
+	var/hijacked = 0
+	for(var/datum/mind/M in get_owners())
+		if(!ispulsedemon(M.current) || QDELETED(M.current))
+			continue
+		var/mob/living/simple_animal/pulse_demon/demon = M.current
+		hijacked += length(demon.hijacked_apcs)
+	return hijacked >= amount
+
+/datum/objective/pulse_demon/drain
+	name = "Drain Power"
+	var/amount = 500000
+
+/datum/objective/pulse_demon/drain/New()
+	. = ..()
+	amount = rand(1, 10) * 500000
```

Comment:
Any reason why do you set the var/amount value only to override it on New()?

Reply:
That's how it was on /vg/, though I personally I don't see anything wrong with a sensible default rather than leaving it null or 0. Do you want it changed to either of those?

Reply:
I personally think its better to change (for hijack objective too), it having a value makes me think it's being used somewhere in the code.

Reply:
I see, it's left as null before being initialized in the latest commit.